<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellModWidget</class>
 <widget class="QWidget" name="CellModWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="trajecCheck">
     <property name="text">
      <string>Operate on trajectory</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="wrapButton">
     <property name="text">
      <string>Wrap atoms</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="cropButton">
     <property name="text">
      <string>Crop atoms</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="multiplyFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="xMultLabel">
        <property name="text">
         <string>&amp;x:</string>
        </property>
        <property name="buddy">
         <cstring>xMultSel</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="xMultSel">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="yMultLabel">
        <property name="text">
         <string>&amp;y:</string>
        </property>
        <property name="buddy">
         <cstring>yMultSel</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="yMultSel">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="zMultLabel">
        <property name="text">
         <string>&amp;z:</string>
        </property>
        <property name="buddy">
         <cstring>zMultSel</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QSpinBox" name="zMultSel">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QPushButton" name="multButton">
        <property name="text">
         <string>Multiply cell</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QFrame" name="alignFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QComboBox" name="stepVecSel">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="coordVecSel">
        <item>
         <property name="text">
          <string>x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>z</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="alignButton">
        <property name="text">
         <string>Align cell</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="stepVecLabel">
        <property name="text">
         <string>Cell &amp;vector:</string>
        </property>
        <property name="buddy">
         <cstring>stepVecSel</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="coordVecLabel">
        <property name="text">
         <string>&amp;Real vector:</string>
        </property>
        <property name="buddy">
         <cstring>coordVecSel</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QFrame" name="reshapeFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="cdmSel">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="reshapeLabel">
        <property name="text">
         <string>&amp;New Cell:</string>
        </property>
        <property name="buddy">
         <cstring>cdmSel</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="cdmFmtSel">
        <item>
         <property name="text">
          <string>Angstrom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bohr</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QTableWidget" name="reshapeTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>0</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>x</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>y</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>z</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="2" column="2">
         <property name="text">
          <string>1</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QPushButton" name="reshapeButton">
        <property name="text">
         <string>Reshape cell</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
